ts/features/bonus/cgn/store/reducers/details.ts

```diff
@@ -45,7 +46,19 @@ export default reducer;
 export const cgnDetailSelector = (state: GlobalState) =>
   state.bonus.cgn.detail.information;
 
-export const isCgnActive = createSelector(
+// Returns true only if card informations are available and not in PENDING status
+export const isCgnInformationAvailableSelector = createSelector(
   cgnDetailSelector,
-  (information: pot.Pot<Card, NetworkError>): boolean => pot.isSome(information)
+  (information: pot.Pot<Card, NetworkError>): boolean =>
+    pot.isSome(information) && !CardPending.is(information)
```

Comment:
```suggestion
    pot.isSome(information) && !CardPending.is(information.value)
```

if I understand correcly, the second condition should be `!CardPending.is(information.value)`? Because information is a `Pot` type and it would be always `true`.